src/dso_api/dynamic_api/remote/serializers.py

```diff
@@ -63,11 +96,25 @@ def get_schema(self, instance):
         dataset_path = Dataset.objects.get(name=name).path
         return f"https://schemas.data.amsterdam.nl/datasets/{dataset_path}/dataset#{table}"
 
+    def _lookup_field(self, id):
+        return self.table_schema.get_field_by_id(id)
+
+
+class RemoteFieldSerializer(DSOSerializer, _AuthMixin):
```

Comment:
Als je `AuthMixin` vooraan zet, kan je wel `get_fields()` herimplementeren in de mixin.

Reply:
Was ik ook achter gekomen, maar mijn motto qua multiple inheritance is "mixins or bust". Liever een paar regels extra code dan wéér moeten uitzoeken hoe de MRO werkt en het daarna weer vergeten.

Reply:
Dan is het trouwens ook geen mixin meer :)